charts/apisix-dashboard/values.yaml

```diff
@@ -24,7 +24,7 @@ image:
   repository: apache/apisix-dashboard
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "2.3"
+  tag: "2.6"
```

Comment:
Also update the verison of chart in Chart.yaml.

Reply:
Updated.